app/Models/Field.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Field extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['venue_id', 'slug', 'name', 'image', 'description', 'active'];
+
+    public function venue(): BelongsTo
+    {
+        return $this->belongsTo(Venue::class);
+    }
+    public function specification(): BelongsToMany
```

Comment:
pake hasMany atau belongsToMany nya ris?

Reply:
mun ek approve kmh sal @muhamadarsaludin :laughing:

Reply:
I think field relation with field specification, not direct with specification @sejutaimpian

Reply:
You can write like this


public funtion fieldSpecitication()
{
      return $this->hasMany(FiledSpecification:class)
}